public/app/features/dashboard-scene/inspect/HelpWizard/randomizer.ts

```diff
@@ -0,0 +1,77 @@
+import { DataFrameJSON, Labels, FieldType } from '@grafana/data';
+
+export function newLetterRandomizer(): (v: string) => string {
+  const upper = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+  const lower = 'abcdefghijklmnopqrstuvwxyz';
+  const charactersLength = upper.length;
+
+  const history = new Map<string, string>();
+  return (v: string) => {
+    const old = history.get(v);
+    if (old != null) {
+      return old;
+    }
+    const r = [...v]
+      .map((c) => {
```

Comment:
This entire file feels like unnecessarily duplicated. Any chance we could remove the old-arch version of it?

Reply:
Yes, I checked and the content is the same. Removed the old-arch one